### ISSUE TITLE
Setting 'inline-block' and 'relative' position on buttons with custom element

### ISSUE THREAD

**Issue by jeffpamer** [openseadragon/openseadragon#1567]
Control buttons always receive `display: inline-block` and `position: relative` regardless of whether an ID to a custom button has been provided:
https://github.com/openseadragon/openseadragon/blob/81a634a61845aac973cf6b7d00ee0dcbbaf968ea/src/button.js#L195

This potentially breaks the layout of custom styled button elements. If the user is specifying their own button elements, I would think the appropriate behavior would be to _not_ apply any style changes.

**Comment by iangilman** [comment]
Yes, good point. Looks like if no custom button is provided, it already sets the `position: relative`. We'd want to add the `display: inline-block` to that area. 

My only concern is finding a good migration path. It's possible people's code already assumes those styles are getting added to their custom buttons, so removing this will break them (a little bit). One option would be to add those styles unless a special parameter is given. That said, it would be nice to eventually move to this being the default, so I guess I'd love it if we could have a release where it's deprecated somehow and then the next one can have the change for real. What do you think?

**Comment by jeffpamer** [comment]
Yep, that's an understandable concern. I think phasing the current behavior out over a release or two is a great idea (maybe with a deprecation warning logged to the console?). At the moment we're just working around this by selecting the elements and unsetting the display/position styles after init. Which isn't great, but it's far from critical.

**Comment by iangilman** [comment]
Cool. Well, if you're up for making a patch for OSD, that would be lovely :)
